sandbox/src/mocks/availableActionsForUserList/WithEntries.json

```diff
@@ -0,0 +1,48 @@
+{
+	"id": "604dc1ea-986c-45ec-b36c-cf9d2fabc5c0",
+	"meta": {
+		"lastUpdated": "2021-11-01T11:05:40.046Z",
+		"profile": [
+			"http://hl7.org/fhir/STU3/StructureDefinition/Bundle"
+		]
+	},
+	"resourceType": "Bundle",
+	"type": "searchset",
+	"link": [
+		{
+			"relation": "self",
+			"url": "_baseUrl_/STU3/Task?focus=ReferralRequest/000000070000/_history/6&intent=proposal&status=ready"
```

Comment:
I think the first query parameter here should be prefixed with a `?` rather than a `&`.